index.html

```diff
@@ -122,6 +159,161 @@ <h2>Methods</h2>
+                <li>If <var>ext</var>'s "direction" value is not "sendrecv", and "uri"
+                  indicates a mandatory-to-use attribute, throw an IllegalArgument error.
+                </li>
+                <li>if <var>ext</var>'s "direction" value is "stopped", and "uri" indicates
+                  a mandatory-to-implement extension, throw an IllegalArgument error.
+                </li>
+              </ol>
+            </li>
+            <li>
+              For each mandatory-to-implement extension <var>mext</var>, if <var>mext</var>
+              is not present in <var>extlist</var>, throw an IllegalArgument error.
+            </li>
+            <li>
+              Set the [[\HeaderExtensionsToOffer]] to <var>extlist</var>.
+            </li>
+          </ol>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>HeaderExtensionsNegotiated, readonly</dt>
+        <dd>Returns the value of the [[\HeaderExtensionsNegotiated]] slot.</dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+      <p>
+        In the <a>final steps to create an offer</a>, add a step:
+      </p>
+      <ol>
+        <li>If the [[\HeaderExtensionsToOffer]] internal slot is set, for
+          each extension in [[\HeaderExtensionsToOffer]], add an appropriate
+          "a=hdrext" line.
+        </li>
+      </ol>
+      <p>
+        In the <a>set an RTCSessionDescription</a> algorithm, add a step
+        right after the step that sets transceiver.[[\Sender]].[[\SendCodecs]],
+        saying "For each transciever, set [[\HeaderExtensionsNegotiated]] to
+        an empty list, and then
+        for each "a=hdrext" line, add an appropriate
+        RTCRtpHeaderExtensionCapability to the list.
+      </p>
+    </section>
+  </section>
+  <section id="rtcrtpheaderextensionparameters-interface">
+    <h3>
+      <dfn>RTCRtpHeaderExtensionParameters</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtpheaderextensionparameters">RTCRtpHeaderExtensionParameters</a></code>
+      dictionary is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional member.
+    </p>
+    <pre class="idl">
+      partial dictionary RTCRtpHeaderExtensionParameters {
+        boolean enabled;
+      };
+    </pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>enabled, of type boolean</dt>
+        <dd>
+          <p>When returned from getParameters(), true when the RTP sender is
+            configured to send this header extension when appropriate. If the
+            attribute is missing, it means that it cannot be disabled.
+          </p>
+          <p>
+            When passed to setParameters(), true when the RTP sender should
+            send this header extension when appropriate; false when the RTP
+            sender should never send this extension.
```

Comment:
Should this be settable? If the header extension is not negotiated and we pass "true" this needs to throw.

Reply:
Only negotiated extensions are returned from getParameters(). The URL field is read-only, so we throw if it is changed. Need to say that setParameters() throws if the length of the list is different, too.